package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "grunt-vueify",
-  "version": "1.0.1",
+  "version": "1.1.0",
```

```suggestion
  "version": "1.0.1",
```
Please don't bump the version in a merge request. That's a publishing detail that should be unrelated to the changes you want to merge. :)